com.unity.netcode.gameobjects/Runtime/Serialization/ByteUnpacker.cs

```diff
@@ -356,11 +342,17 @@ public static unsafe void ReadValueBitPacked(FastBufferReader reader, out ushort
                     *ptr = *data;
                     *(ptr + 1) = *(data + 1);
                     break;
+                case 3:
+                    // First byte contains no data, it's just a marker. The data is in the remaining two bytes.
+                    *ptr = *(data + 1);
```

Comment:
same as above `*(ptr++) = *(data++);` twice, with extra `++data` at beginning ? </suggestion> maybe it's just style.